include/zephyr/dt-bindings/input/input-event-codes.h

```diff
@@ -230,6 +230,7 @@
  * @anchor INPUT_ABS_CODES
  * @{
  */
+#define INPUT_ABS_MT_SLOT 0x0b          /**< Absolute multitouch slot identifier */
```

Comment:
Can you use the same value that Linux uses (0x2f) and keep this block sorted alphabetically?

Reply:
Done.